solutions/swb-reference/integration-tests/tests/isolated/environments/list.test.ts

```diff
@@ -193,9 +197,37 @@ describe('list environments', () => {
         );
       }
     });
+
+    test('list project environments when user not associated to a valid project', async () => {
+      console.log('A Project Admin of Project1 and Project3, cannot LIST environments from Project2');
+      try {
+        await paSession.resources.projects.project(projectId2).environments().listProjectEnvironments();
```

Comment:
I know we don't do this in all our tests, but we might want to consider throwing an error on line 205. An example is shown [here](https://github.com/aws-solutions/research-service-workbench-on-aws/blob/d035cc34c555b521cc50e5aab88924e5c39c2fa1/solutions/swb-reference/integration-tests/tests/multiStep/costCenter.test.ts#L38)

This protects against false negative where line 204 doesn't throw any error and the test passes.

Reply:
Not a bad idea!